backend/package-lock.json

```diff
@@ -1,9 +1,12 @@
 {
   "name": "backend",
+  "version": "1.0.0",
```

Comment:
What was the reason for these insertions? I'm not saying it's wrong — just curious.

Reply:
I am not entirely sure how that got inserted. I've omitted in my remote repo